ckanext/canada/schemas/presets.yaml

```diff
@@ -2755,25 +2752,64 @@ presets:
         fr: Séparé de
       value: separated_from
     - label:
-        en: Split into ...
-        fr: Divisé en …
+        en: Split into
+        fr: Divisé en
       value: split_into
     - label:
-        en: Merged with ...
+        en: Merged with
         fr: Fusionné avec
       value: merged_into
     - label:
         en: Changed back to
         fr: Restauré à
       value: changed_back_to
+    form_include_blank_choice: true
     form_snippet: select.html
     display_snippet: select.html
     validators: scheming_required scheming_choices
+    required: true
+    form_panel: relationships
+
+# Field = Related Record URL.
+- preset_name: canada_dataset_related_url
+  values:
+    field_name: related_url_translated
```

Comment:
the `_translated` bit was added for fluent versions of core fields, it's not necessary for new custom fields (and it's a little odd to call urls translated)

Reply:
I guess we might not need to have a URL (English) and URL(French) then? otherwise the `_translated` is really just for the get_translated helper.

Reply:
There will be cases where users need to provide separate EN and FR urls so I don't think we can get rid of that. Yes the `_translated` is just for `get_translated` which isn't used for custom fields, only core fields.

Reply:
ah okay I see, `_translated` has been removed.